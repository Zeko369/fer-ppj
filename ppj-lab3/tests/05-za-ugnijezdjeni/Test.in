<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 x
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 2
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 k
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 0
       <T_lista>
        $
      <E_lista>
       $
   <lista_naredbi>
    <naredba>
     <za_petlja>
      KR_ZA 3 za
      IDN 3 i
      KR_OD 3 od
      <E>
       <T>
        <P>
         BROJ 3 1
        <T_lista>
         $
       <E_lista>
        $
      KR_DO 3 do
      <E>
       <T>
        <P>
         BROJ 3 5
        <T_lista>
         $
       <E_lista>
        $
      <lista_naredbi>
       <naredba>
        <naredba_pridruzivanja>
         IDN 4 x
         OP_PRIDRUZI 4 =
         <E>
          <T>
           <P>
            IDN 4 x
           <T_lista>
            OP_PUTA 4 *
            <T>
             <P>
              IDN 4 i
             <T_lista>
              $
          <E_lista>
           $
       <lista_naredbi>
        <naredba>
         <za_petlja>
          KR_ZA 5 za
          IDN 5 j
          KR_OD 5 od
          <E>
           <T>
            <P>
             BROJ 5 5
            <T_lista>
             $
           <E_lista>
            $
          KR_DO 5 do
          <E>
           <T>
            <P>
             BROJ 5 8
            <T_lista>
             $
           <E_lista>
            $
          <lista_naredbi>
           <naredba>
            <naredba_pridruzivanja>
             IDN 6 k
             OP_PRIDRUZI 6 =
             <E>
              <T>
               <P>
                IDN 6 x
               <T_lista>
                $
              <E_lista>
               OP_PLUS 6 +
               <E>
                <T>
                 <P>
                  BROJ 6 3
                 <T_lista>
                  $
                <E_lista>
                 $
           <lista_naredbi>
            $
          KR_AZ 7 az
        <lista_naredbi>
         $
      KR_AZ 8 az
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 9 k
       OP_PRIDRUZI 9 =
       <E>
        <T>
         <P>
          IDN 9 k
         <T_lista>
          $
        <E_lista>
         OP_PLUS 9 +
         <E>
          <T>
           <P>
            BROJ 9 1
           <T_lista>
            $
          <E_lista>
           $
     <lista_naredbi>
      $
