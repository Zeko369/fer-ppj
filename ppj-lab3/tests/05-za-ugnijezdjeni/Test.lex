IDN 1 x
OP_PRIDRUZI 1 =
BROJ 1 2
IDN 2 k
OP_PRIDRUZI 2 =
BROJ 2 0
KR_ZA 3 za
IDN 3 i
KR_OD 3 od
BROJ 3 1
KR_DO 3 do
BROJ 3 5
IDN 4 x
OP_PRIDRUZI 4 =
IDN 4 x
OP_PUTA 4 *
IDN 4 i
KR_ZA 5 za
IDN 5 j
KR_OD 5 od
BROJ 5 5
KR_DO 5 do
BROJ 5 8
IDN 6 k
OP_PRIDRUZI 6 =
IDN 6 x
OP_PLUS 6 +
BROJ 6 3
KR_AZ 7 az
KR_AZ 8 az
IDN 9 k
OP_PRIDRUZI 9 =
IDN 9 k
OP_PLUS 9 +
BROJ 9 1
