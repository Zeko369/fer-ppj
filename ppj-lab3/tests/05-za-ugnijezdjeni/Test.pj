x = 2 
k = 0
za i od 1 do 5
 x = x * i
 za j od 5 do 8
  k = x + 3
 az
az
k = k + 1
