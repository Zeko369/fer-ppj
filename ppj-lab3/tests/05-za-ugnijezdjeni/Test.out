4 1 x
4 3 i
6 1 x
9 2 k
